airflow/providers/yandex/yq_client/query_results.py

```diff
@@ -0,0 +1,331 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The `yq_client` has many functions but no test coverage.
It's odd that we need so many helpers functions. Providers are suppose to be very slim in code.

I think the issue here is that this PR adds to Airflow capabilities that should be in Yandex Python SDK ?

Reply:
NVM I see discussion around it in https://github.com/apache/airflow/pull/37458#discussion_r1491375103

Reply:
If you don't mind I will resolve this issue. Reopen if you have any question. Thank you for feedback!